### PR TITLE
chore(cli): copilot renaming for app status

### PR DESCRIPTION
**Pull request by iamhopaul123**
Copilot renaming for `app status`. Also complete renaming for `describe/status.go` that was left behind. Partially address #908.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: internal/pkg/cli/svc_status.go [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/cli/selector"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/config"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/describe"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/color"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/log"
+	"github.com/spf13/cobra"
+)
+
+const (
+	svcStatusAppNamePrompt     = "Which application's services status would you like to show?"
```
Comment (kohidave) [inline internal/pkg/cli/svc_status.go:19]: > Which app is the service in? 

What do you think of ^

File: internal/pkg/cli/svc_logs.go [hunk 2]
```diff
@@ -251,10 +251,9 @@ func (o *svcLogsOpts) askSvcEnvName() error {
 
 	// return if only one deployed service found
 	if len(svcEnvNames) == 1 {
-		log.Infof("Only found one deployed service, defaulting to: %s\n", color.HighlightUserInput(svcEnvNames[0]))
 		o.svcName = svcEnvs[svcEnvNames[0]].svcName
 		o.envName = svcEnvs[svcEnvNames[0]].envName
-
+		log.Infof("Show logs of service %s deployed in environment %s\n", color.HighlightUserInput(o.svcName), color.HighlightUserInput(o.envName))
```
Comment (efekarakus) [inline internal/pkg/cli/svc_logs.go:256]: Maybe?
```suggestion
		log.Infof("Showing logs of service %s deployed in environment %s\n", color.HighlightUserInput(o.svcName), color.HighlightUserInput(o.envName))
```

File: internal/pkg/cli/svc_status.go [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+	svcStatusAppNameHelpPrompt = "An application groups all of your services together."
+	svcStatusNamePrompt        = "Which service's status would you like to show?"
+	svcStatusNameHelpPrompt    = "Displays the service, tasks and CloudWatch alarms status."
```
Comment (kohidave) [inline internal/pkg/cli/svc_status.go:22]: > Displays the service's task status, most recent deployment and alarm statuses. 

^ What do you think of that?

File: internal/pkg/cli/svc_status.go [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+
+	return &svcStatusOpts{
+		svcStatusVars: vars,
+		store:         store,
+		w:             log.OutputWriter,
+		sel:           selector.NewConfigSelect(vars.prompt, store),
+		initSvcDescriber: func(o *svcStatusOpts, envName, svcName string) error {
+			d, err := describe.NewServiceDescriber(o.AppName(), envName, svcName)
+			if err != nil {
+				return fmt.Errorf("creating stack describer for application %s: %w", o.AppName(), err)
```
Comment (efekarakus) [inline internal/pkg/cli/svc_status.go:58]: Should this be?
```suggestion
				return fmt.Errorf("creating service describer for application %s, environment %s, and service %s: %w", o.AppName(), envName, svcName, err)
```

Comment (iamhopaul123) [reply internal/pkg/cli/svc_status.go:58]: done!

File: internal/pkg/cli/svc_logs.go (hunk 2 above)
Comment (iamhopaul123) [reply internal/pkg/cli/svc_logs.go:256]: done!

**Pull request merged**